WhiteDentalClinic/WhiteDentalClinic.Application/Models/MedicalServiceModel/UpdateRequestMedicalService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace WhiteDentalClinic.Application.Models.MedicalServiceModel
+{
+    public class UpdateRequestMedicalService
+    {
+        public int Price { get; set; }
```

Comment:
In my opinion the name of the medical service should be updatable as well. I think is happens from type to time to "upgrade" to fancier terms for their names.